maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PrepareReleaseMojo.java

```diff
@@ -455,7 +454,12 @@ private String getLineSeparatorFromPom()
     {
         char current;
         String lineSeparator = "";
-        try ( InputStream is = new FileInputStream( this.project.getFile() ) )
+        File releasePom = this.project.getFile();
+        if (releasePom == null)
```

Comment:
Can this file be `null` during prepare? How?

Reply:
not during prepare but it was null within some tests.